src/main/webapp/app/entities/quiz/short-answer-question.model.ts

```diff
@@ -7,6 +7,9 @@ export class ShortAnswerQuestion extends QuizQuestion {
     public spots?: ShortAnswerSpot[];
     public solutions?: ShortAnswerSolution[];
     public correctMappings?: ShortAnswerMapping[];
+    public matchLetterCase: Boolean;
+    public similarityValue: Number = 100;
```

Comment:
Add a comment that those are the default values in the client.
I think we should align them with the ones on the server, so haven false and 85 here as well.
(I know this is contrary to my previous comment about this, but I think it would be more consistent)